builtin/logical/consul/backend_test.go

```diff
@@ -99,7 +99,14 @@ func TestBackend_Renew_Revoke(t *testing.T) {
 				testBackendRenewRevoke(t, "1.4.4")
 			})
 
-			testBackendRenewRevoke14(t, "")
+			t.Run("param-policies", func(t *testing.T) {
+				t.Parallel()
+				testBackendRenewRevoke14(t, "", "policies")
+			})
+			t.Run("param-consul_policies", func(t *testing.T) {
+				t.Parallel()
+				testBackendRenewRevoke14(t, "", "consul_policies")
+			})
```

Comment:
@fairclothjm Thanks, I've added a split from this single test to now run with each parameter.

Reply:
Can we add a test case to verify what happens when we set both `policies` and `consul_policies` within the same request?

Reply:
Took me a bit on how I could verify that both are being received and that `consul_policies` was not taking on the value from `policies` but I think I made a working solution.